components/pages/HomePage/index.tsx

```diff
@@ -349,6 +349,15 @@ export default class HomePage extends React.Component<IndexProps, IndexState> {
             />
           </ScrollView>
           <WPFooter base={homePosts} />
+          <div
+            dangerouslySetInnerHTML={{
+              __html: `<script type="application/ld+json">${JSON.stringify({
```

Comment:
This library would probably be a cleaner alternative: https://github.com/google/react-schemaorg

Reply:
Seems like a good thing to explore for tech debt work, but I think we're good merging this as is since it's urgent?